github-api-integration-module.sh

```
@@ -0,0 +1,8 @@
+#this is the exmple of shell scripts
+#!/bin/bash
```

Shellcheck (0.10.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)